src/player.ts

```diff
@@ -491,4 +496,42 @@ export class Player extends NativeInstance<PlayerConfig> {
   getThumbnail = async (time: number): Promise<Thumbnail | null> => {
     return PlayerModule.getThumbnail(this.nativeId, time);
   };
+
+  /**
+   * Whether casting to a cast-compatible remote device is available. `CastAvailableEvent` signals when
+   * casting becomes available.
+   *
+   * @platform iOS, Android
+   */
+  isCastAvailable = async (): Promise<boolean> => {
+    return PlayerModule.isCastAvailable();
+  };
+
+  /**
+   * Whether video is currently being casted to a remote device and not played locally.
+   *
+   * @platform iOS, Android
+   */
+  isCasting = async (): Promise<boolean> => {
+    return PlayerModule.isCasting(this.nativeId);
+  };
+
+  /**
+   * Initiates casting the current video to a cast-compatible remote device. The user has to choose to which device it
+   * should be sent.
+   *
+   * @platform iOS, Android
+   */
+  castVideo = () => {
+    PlayerModule.castVideo(this.nativeId);
+  };
+
+  /**
+   * Stops casting the current video. Has no effect if [isCasting] is false.
```

Comment:
```suggestion
   * Stops casting the current video. Has no effect if `isCasting` is false.
```
I don't think `[]` linking is supported in typescript

Reply:
Fixed in 47a8071